packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -200,6 +202,7 @@ export interface StatefulCallClient extends CallClient {
     createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
     createView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState, options?: CreateViewOptions): Promise<CreateViewResult | undefined>;
     disposeView(callId: string | undefined, participantId: CommunicationIdentifier | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState): void;
+    getEnvironmentInfo(): Promise<EnvironmentInfo>;
```

Comment:
cc @PorterNan @prprabhu-ms should we create methods off the stateful client interface, or should we proxy the `features(Feature.DebugInfo)` instead?

Reply:
(although in its current state this method is redundant, see comment below)

Reply:
I think given we don't want two methods that do the same-but-different things, we should be proxying the features fn instead of defining a new method here

Reply:
Will update this!